data/json/recipes/recipe_appliance.json

```diff
@@ -106,6 +106,38 @@
     "using": [ [ "soldering_standard", 5 ] ],
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "proficiencies": [ { "proficiency": "prof_elec_soldering" } ],
-    "components": [ [ [ "frame_wood", 1 ] ], [ [ "battery_charger", 1 ] ] ]
+    "components": [ [ [ "frame_wood", 1 ] ], [ [ "battery_charger", 1 ] ], [ [ "medium_battery_cell", 1 ] ] ]
+  },
+  {
+    "result": "portable_air_conditioner",
```

Comment:
Same retrofiting vehicle AC sounds pretty weird, and should at best only be a secondary way to get a cooler working